test/Fortran/SHF95FLAGS.py

```diff
@@ -75,29 +75,30 @@
 test.must_match(obj_ + 'test13' + _obj, " -c -x\nThis is a .f95 file.\n")
 test.must_match(obj_ + 'test14' + _obj, " -c -x\nThis is a .F95 file.\n")
 
-
-
 fc = 'f95'
 g95 = test.detect_tool(fc)
-
 if g95:
-
     test.subdir('x')
-
     test.write(['x','dummy.i'],
 """
 # Exists only such that -Ix finds the directory...
 """)
 
+    # ref: test/fixture/wrapper.py
     test.file_fixture('wrapper.py')
-
     test.write('SConstruct', """
-foo = Environment(SHF95 = '%(fc)s')
+foo = Environment(SHF95='%(fc)s')
 shf95 = foo.Dictionary('SHF95')
-bar = foo.Clone(SHF95 = r'%(_python_)s wrapper.py ' + shf95)
-bar.Append(SHF95FLAGS = '-Ix')
-foo.SharedLibrary(target = 'foo/foo', source = 'foo.f95')
-bar.SharedLibrary(target = 'bar/bar', source = 'bar.f95')
+#print(f"foo SHF95={foo.Dictionary('SHF95')}", file=sys.stderr)
```

Comment:
Debug? Can remove?

Reply:
okay, not the emoji I was looking for.  That was definitely not supposed to be in there, was sure I'd cleaned it out.